masschallenge/django-accelerator
# [AC-4598] New build environment

I recommend that you start with the PR for impact-api.

Work done:
* Complete revision of the Makefile and associated targets.
• All regular developer workflows up to release/deploy should be supported by make targets.

To test:
• Run `make help` to understand what the targets are and should do
• Run `make test` to make sure everything is working.
• Review all the other targets.
